mockgen/mockgen.go

```diff
@@ -766,16 +766,30 @@ func (g *generator) getArgNames(m *model.Method, in bool) []string {
 		params = m.Out
 	}
 	argNames := make([]string, len(params))
+
+	// check if the name already exists as an imported package name
+	// which would cause a name collision
+	nameExistsAsPackage := func(name string) bool {
+		for _, symbolName := range g.packageMap {
+			if symbolName == name {
+				return true
+			}
+		}
+		return false
+	}
```

Comment:
Is there a reason this needs to be defined as a closure here, instead of a method on generator type? It doesn't seem like it's capturing any value here, so I think this could be defined as a helper method.

Reply:
Nope, no specific reason. I will change that right now :)